distribution/openrct2.d.ts

```diff
@@ -356,7 +356,7 @@ declare global {
     }
 
     type TileElementType =
-        "surface" | "footpath" | "track" | "small_scenery" | "wall" | "entrance" | "large_scenery" | "banner";
+        "surface" | "footpath" | "track" | "small_scenery" | "wall" | "entrance" | "large_scenery" | "banner" | "openrct2_corrupt";
```

Comment:
```suggestion
        "banner" | "entrance" | "footpath" | "large_scenery" | "openrct2_corrupt"| "small_scenery" |"surface" | "track" | "wall";
```
Can we take advantage of this change and also add another commit that sorts this list? :D

Reply:
Might as well do that for all other lists. 😄 Currently, they are sorted by their type ID though, which might be nice to keep too.